stages/org.osbuild.zipl.inst

```diff
@@ -85,27 +85,30 @@ def find_kernel(root, kernel: str):
 def main(paths, devices, options):
     kver = options["kernel"]
     location = options["location"]
-    blocksize = options.get("blocksize", 512)
 
     root = paths["mounts"]
     dev = paths["devices"]
 
     disk = devices["disk"]
     device = os.path.join(dev, disk["path"])
-
-    kernel, initrd, kopts = find_kernel(root, kver)
-
-    subprocess.run(["/usr/sbin/zipl",
-                    "--verbose",
-                    "--target", f"{root}/boot",
-                    "--image", kernel,
-                    "--ramdisk", initrd,
-                    "--parameters", kopts,
-                    "--targetbase", device,
-                    "--targettype", "SCSI",
-                    "--targetblocksize", str(blocksize),
-                    "--targetoffset", str(location)],
-                   check=True)
+    try:
+        subprocess.run(["/usr/sbin/kpartx", "-l", "-f", "-a", "-v", "-s", device], check=True)
+        loopback = os.path.basename(device)
+        subprocess.run(["mount", f"/dev/mapper/{loopback}p2", root], check=True)
+        subprocess.run(["mount", f"/dev/mapper/{loopback}p1", f"{root}/boot"], check=True)
```

Comment:
I don't think you'll need all this mount stuff here if you do something like https://github.com/osbuild/osbuild/pull/1551/files#r1464006813